client/src/main/java/cloud/prefab/client/internal/ConfigClientImpl.java

```diff
@@ -323,6 +345,56 @@ public void onCompleted() {
       );
   }
 
+  private void startLogStatsUploadExecutor() {
+    ScheduledExecutorService scheduledExecutorService = Executors.newScheduledThreadPool(
+      1,
+      r -> new Thread(r, "prefab-logger-stats-uploader")
```

Comment:
We probably want to mark this thread as daemon so that it doesn't block shutdown